statistics/handle/handle.go

```diff
@@ -251,28 +267,47 @@ func (h *Handle) updateStatsCache(newCache statsCache) {
 	h.statsCache.Lock()
 	oldCache := h.statsCache.Load().(statsCache)
 	if oldCache.version <= newCache.version {
+		h.statsCache.memTracker.Consume(newCache.memUsage - oldCache.memUsage)
```

Comment:
besides minus the oldCache.memUsage, the newCache.memUsage should also be added?

Reply:
newCache.memUsage is up-to-date before invoke updateStatsCache().
`newCache` is used to update handle.StateCache, after memTracker.Consume()，it will be stored in h.StateCache.